### DISCUSSION TITLE
How can a boss use both superjump and teleport?

### DISCUSSION THREAD

**Discussion by naydef** [Batfoxkid/Freak-Fortress-2-Rewrite#92] [Q&A]
`special_mobility` contains both super jump and teleportation, albeit the old way of doing it by adding suffix to the ability name does not work currently.

So how it be done here?

**Comment by Batfoxkid** [comment]
By setting the "options" to 3, you can use both Super Jump and Teleport, in this case it use shared cooldowns and charge rates but Super Jump is available while not fully charged and Teleport is available while fully charged.
```
	"special_mobility"
	{
		"slot"			"1"						// Charge slot (Only used for sound_ability)
		"options"		"3"						// Mobility flags (1=Super Jump, 2=Teleport)
		"button"		"11"					// Button type (11=M2, 13=Reload, 25=M3)
		"charge"		"1.5"					// Time to fully charge
		"cooldown"		"5.0"					// Cooldown after use
		"delay"			"5.0"					// Delay before first use
		"upward"		"750 + (n * 3.25)"		// Super Jump upward velocity set (n=0.0 ~ 100.0)
		"forward"		"1.0 + (n * 0.000275)"	// Super Jump forward velocity multi (n=0.0 ~ 100.0)
		"emergency"		"2000.0"				// Super Jump upward velocity added when touching a hazard
		"stun"			"2.0"					// Teleport stun duration
		
		"plugin_name"	"ff2r_default_abilities"
	}
```

**Reply by naydef** [reply]
Hm, I see. However the boss which is being converted has different cooldowns for jump and teleport. I guess it's not possible currently to different parameters.

Maybe we'll go use the old abilities.
